### PR TITLE
Gui 2814 Added elb logging and create redirect.

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-2814

### CODE REVIEW

File: eucaconsole/static/js/pages/elb-wizard/advanced.js [hunk 1]
```diff
@@ -1,16 +1,66 @@
...
+    };
+    $scope.$watch('advanced.values.bucketName', function(newVal, oldVal) {
+        if (newVal === oldVal) return;
+        vm.buckets.push(newVal);
+    });
+}])
+.directive('loggingConfirmDialog', function() {
+    return {
+        restrict: 'A',
+        require: ['^modal', 'loggingConfirmDialog'],
```
Comment (coyote240) [inline eucaconsole/static/js/pages/elb-wizard/advanced.js]: Do you need the require param if you're not using a link function?

File: eucaconsole/static/js/pages/elb-wizard/advanced.js [hunk 2]
```diff
@@ -1,16 +1,66 @@
 angular.module('ELBWizard')
-.controller('AdvancedController', ['$scope', '$routeParams', 'ELBWizardService', 'ELBService', function ($scope, $routeParams, ELBWizardService, ELBService) {
-    this.values = ELBWizardService.values;
-    this.createELB = function($event) {
+.config(function($locationProvider) {
```
Comment (dkavanagh) [inline eucaconsole/static/js/pages/elb-wizard/advanced.js]: Oh, sure.. yes this should be for the entire wizard.

File: eucaconsole/static/js/widgets/create_bucket_dialog2.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                $event.preventDefault();
+                if ($scope.createBucketForm.$invalid) {
+                    return false;
+                }
+                var formData = {
+                    'csrf_token': $('#csrf_token').val(),
+                    'bucket_name': vm.bucketName
+                };
+                vm.isCreatingBucket = true;
+                $http({
```
Comment (dkavanagh) [inline eucaconsole/static/js/widgets/create_bucket_dialog2.js]: oh, of course. I had created the bucket-service like 2 days ago. not sure why I didn't throw this in there.

File: eucaconsole/static/js/pages/elb-wizard/advanced.js (hunk 1 above)
Comment (dkavanagh) [inline eucaconsole/static/js/pages/elb-wizard/advanced.js]: I guess I didn't think that much about why require was used elsewhere. I was thinking it was just about being stricter about what to expect in the markup. could remove.

**Pull request merged**